### DISCUSSION TITLE
Convert JPEG/PNG to SVG with pallets

### DISCUSSION THREAD

**Discussion by blpraveen** [ImageMagick/ImageMagick#6803] [Help]
### ImageMagick version

7.1.1-11

### Operating system, version and so on

Windows

### Description

I am looking php code  to convert jpeg/png image to svg with rgb collor pallets pass as parameter

**Comment by fmw42** [comment]
Imagemagick may change the color palette to conform to its standards. So it may not be preserved unchanged.  When you say PHP code, do you mean Imagick or something else.  Imagick is not a product of Imagemagick. You should contact the developer of Imagick.  You may get some one here who know that, but that would just be a good gesture.
